### PR TITLE
Update `LICENSE`

### PR DESCRIPTION
**Pull request by avinassh**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,32 @@
+For `status.py` file:
```
Comment (carltongibson) [inline LICENSE:1]: I'd provide a link to the original source here. `... originally from ...`

**Comment by ryansb** [comment]
I accept having my contribution relicensed to comply with DRF licensing.

File: LICENSE (hunk 1 above)
Comment (avinassh) [inline LICENSE:1]: @carltongibson I will do that, thanks!

**Comment by ryansb** [comment]
:+1: looks good to me

**Comment by avinassh** [comment]
Thanks @ryansb :+1:

**Pull request merged**